### PR TITLE
Make Google import more consistent

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/views/register/setup.coffee [hunk 1]
```diff
@@ -83,28 +100,39 @@ module.exports = class RegisterSetupView extends Mn.ItemView
...
-            args = [getProgress, timer.toProperty()]
-            if 'contacts' in imports
-                args.push fromSocket('contacts').toProperty()
-            if 'calendars' in imports
-                args.push fromSocket('calendars').toProperty()
+            streams = imports.map (datasource) ->
+                fromSocket(datasource).toProperty()
 
+            args = [getProgress, timer()].concat streams
             @progress = Bacon.combineWith.apply Bacon, args
```
Comment (aenario) [inline client/app/views/register/setup.coffee]: Bacon.combineWith getProgress, timer(), streams...

**Comment by aenario** [comment]
There also appears to be some conflicts with locales following commit 9a23647, could you take a look ?

File: client/app/views/register/setup.coffee (hunk 1 above)
Comment (m4dz) [inline client/app/views/register/setup.coffee]: Oooohhhh :heart_eyes:

**Comment by m4dz** [comment]
@aenario it's ok for me :+1:

**Pull request merged**